src/app/payments/get-protocol-fee.ts

```diff
@@ -0,0 +1,129 @@
+import { decodeInvoice } from "@domain/bitcoin/lightning"
```

Comment:
just for info, seen the filename I was expecting that `payments/protocol-fee` would also cover the onchain fees.